### PR TITLE
AllowGroups must be an object

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/6075

### CODE REVIEW

**Comment by stephdl** [comment]
the AllowGroups must be an object because we use the `keys`

https://github.com/NethServer/nethserver-cockpit/blob/master/api/system-openssh/update#L45

File: api/system-openssh/update [hunk 1]
```diff
@@ -42,19 +42,16 @@ foreach my $prop (qw(
...
+foreach my $g (keys $obj) {
+    if ($obj->{$g} eq 'ssh') {
+        $AllowGroups .= "$g,";
+    } elsif ($obj->{$g} eq 'sftp') {
+        $AllowGroups .= "$g:sftp,";
+    }
 }
-
+chop $AllowGroups;
+$cdb->set_prop('sshd', 'AllowGroups', $AllowGroups) if(ref($data->{'props'}{'AllowGroups'}) eq 'HASH');
```
Comment (DavidePrincipi) [inline api/system-openssh/update]:
```suggestion
$cdb->set_prop('sshd', 'AllowGroups', $AllowGroups) if(ref($obj) eq 'HASH');
```

Comment (stephdl) [reply api/system-openssh/update]: it is merged, do you want to open a new PR ?

Comment (DavidePrincipi) [reply api/system-openssh/update]: I already pushed the commit to master :innocent:

Comment (stephdl) [reply api/system-openssh/update]: super

**Pull request merged**